app/View/Composers/App.php

```diff
@@ -20,32 +20,32 @@ class App extends Composer
      *
      * @return array
      */
-  public function with()
-  {
-      return [
-          'siteLogo' => $this->siteLogo(),
-      ];
-  }
+    public function with()
+    {
+        return [
+            'siteLogo' => $this->siteLogo(),
+        ];
+    }
 
 
-  public function siteLogo(): string
-  {
-    if (function_exists('get_field')) {
-      $logoStandard = get_field('s_logo', 'option');
-      $logoRetina = get_field('s_logo_retina', 'option');
+    public function siteLogo(): string
+    {
+        if (function_exists('get_field')) {
+            $logo = get_field('site', 'option')['logo'];
```

Comment:
Use the theme settings provider instead. For more info read the docs. `firestarter()->settings()->get('site_logo')`